### PR TITLE
Updated change file

### PR DESCRIPTION
**Pull request by jitendra-webkul**
**BUGS:**

Updated change file

### CODE REVIEW

File: CHANGELOG for v1.x.x.md [hunk 1]
```diff
@@ -2,6 +2,382 @@
 
 #### This changelog consists the bug & security fixes and new features being included in the releases listed below.
 
+## **v1.1.3 (19th of Jine 2020)** - *Release*
```
Comment (ghermans) [inline CHANGELOG for v1.x.x.md:5]: This should be June instead of Jine @jitendra-webkul

File: CHANGELOG for v1.x.x.md [hunk 2]
```diff
@@ -2,6 +2,382 @@
...
+
+* #2982 [fixed] - Column not found: 1054 Unknown column 'default_name' in 'field list' (v1.1.2)
+
+* #2985 [fixed] - Product category is not saving
+
+* #2986 [fixed] - How to enable Vue debug
+
+* #2987 [fixed] - mult address
+
+* #2988 [fixed] - Facebook Customer Chat for Laravel
```
Comment (ghermans) [inline CHANGELOG for v1.x.x.md:144]: Why are general questions being added to this list as "fixed" as it doesn't contain any change in Bagisto?

File: CHANGELOG for v1.x.x.md (hunk 1 above)
Comment (jitendra-webkul) [reply CHANGELOG for v1.x.x.md:5]: Done in this PR #3282

File: CHANGELOG for v1.x.x.md (hunk 2 above)
Comment (jitendra-webkul) [reply CHANGELOG for v1.x.x.md:144]: Yes, you are right, I am removing general questions from change log. Thanks for pointing out

**Pull request merged**